src/core/client.ts

```diff
@@ -255,6 +284,27 @@ export default class Client {
     })
   }
 
+  checkIn(id: string): void {
+    this.checkInAsync(id).then()
```

Comment:
`.then()` is simply there so that the IDE won't complain that the promise result is unhandled.